examples/DAZLS/better_dazls.ipynb

```diff
@@ -0,0 +1,819 @@
+      "R-sqr= 0.30408593867613676\n",
+      "HFDP\n",
+      "RMSE test= 0.29731055878988316\n",
+      "R-sqr= -1.3908466829338044\n",
+      "LLS\n",
+      "RMSE test= 0.18550646112011965\n",
+      "R-sqr= 0.6928684901176348\n",
+      "MNZL\n",
+      "RMSE test= 0.06787005934175006\n",
+      "R-sqr= 0.2971844944391836\n",
+      "NRYN\n",
+      "RMSE test= 0.060716139554765194\n",
+      "R-sqr= 0.692598476068661\n",
+      "OWD\n",
+      "RMSE test= 0.0983913348736813\n",
+      "R-sqr= 0.8999194024661905\n",
+      "WEW\n",
+      "RMSE test= 0.0879411676678399\n",
+      "R-sqr= 0.5557680473477729\n",
+      "WHF\n",
+      "RMSE test= 0.031980482086807246\n",
+      "R-sqr= 0.8101134986300711\n",
+      "WLS\n",
+      "RMSE test= 0.04693656013938843\n",
+      "R-sqr= 0.7355097726581723\n",
+      "WWF\n",
+      "RMSE test= 0.03471334262302751\n",
+      "R-sqr= 0.7062994479576562\n"
+     ]
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from sklearn.metrics import r2_score, mean_squared_error, mean_absolute_error\n",
+    "import glob\n",
+    "import pandas as pd\n",
+    "import os\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.decomposition import FastICA\n",
+    "from sklearn.preprocessing import StandardScaler, MinMaxScaler, MaxAbsScaler, RobustScaler, Normalizer, normalize\n",
+    "from sklearn.linear_model import Lasso, MultiTaskLassoCV\n",
+    "from sklearn.ensemble import RandomForestRegressor, AdaBoostRegressor\n",
+    "from sklearn.neighbors import KNeighborsRegressor\n",
+    "from sklearn.neural_network import MLPRegressor\n",
+    "from sklearn.tree import DecisionTreeRegressor\n",
+    "from sklearn.gaussian_process import GaussianProcessRegressor\n",
+    "from sklearn.decomposition import KernelPCA, PCA\n",
+    "import random\n",
+    "from sklearn.utils import shuffle\n",
+    "from scipy.linalg import fractional_matrix_power\n",
+    "from sklearn.covariance import LedoitWolf, OAS\n",
+    "from sklearn.base import BaseEstimator\n",
+    "\n",
+    "# Seed, path, etc,\n",
+    "random.seed(999)\n",
+    "np.random.seed(999)\n",
+    "\n",
+    "path = os.path.dirname(os.path.abspath(\"prep_data\"))\n",
+    "folder = ['\\\\prep_data\\\\']\n",
+    "combined_data = []\n",
+    "station_name = []\n",
+    "\n",
+    "# Read prepared data\n",
+    "for file_name in glob.glob(path + folder[0] + '*.csv'):\n",
+    "    x = pd.read_csv(file_name, low_memory=False, parse_dates=[\"datetime\"])\n",
+    "    x[\"datetime\"] = pd.to_datetime(x[\"datetime\"])\n",
+    "    x = x.set_index('datetime')\n",
+    "    x.columns=[x.lower() for x in x.columns]\n",
+    "    combined_data.append(x)\n",
+    "    sn = os.path.basename(file_name)\n",
+    "    station_name.append(sn[:len(sn) - 4])\n",
+    "\n",
+    "\n",
+    "# DAZLS algorithm\n",
+    "class DAZLS(BaseEstimator):\n",
```

Comment:
Please do not define the classes here, but import the DAZLS from openstef. You didn't add it to OpenSTEF for it not to be used;)